### PR TITLE
Fix kas container paths

### LINKED ISSUE #138: Paths from kas-container don't resolve well
We do all our builds with kas-container, the commandWrapper and so on work perfectly to get the initial setup but one issue we run into is that there are still paths that get picked up as being /work/... for our layers or /build/...
It doesn't seem to be covered by the support that already exists for dockers, or at least not that I'm aware of.
Is this something I can fix with the current options available or is this under the scope of an improvement?

I'm working through WSL/Remotes if that matters

### PR DESCRIPTION
**Pull request by deribaucourt**
Fixes #138

### CODE REVIEW

File: client/src/lib/src/utils/files.ts [hunk 1]
```diff
@@ -5,6 +5,12 @@
...
 
+export const bitbakeFileExtensions = ['.bb', '.bbappend', '.bbclass', '.conf', '.inc']
+
 export function extractRecipeName (filePath: string): string {
-  return path.parse(filePath).name.split('_')[0]
+  // Ex: gst1.0-plugins-bad_1.18.4.bb -> gst1.0-plugins-bad
+  const baseName = path.parse(filePath).base
+  const fileName = baseName.replace(new RegExp(bitbakeFileExtensions.join('|'), 'g'), '')
+  const recipeName = fileName.split('_')[0]
+  return recipeName
```
Comment (WilsonZiweiWang) [inline client/src/lib/src/utils/files.ts:15]: According to the node [docs](https://nodejs.org/api/path.html#pathextnamepath) on `path`. The example you put should produce the intended extension with the original function logic.

**Review by WilsonZiweiWang — CHANGES_REQUESTED** [review]

File: client/src/lib/src/utils/files.ts (hunk 1 above)
Comment (WilsonZiweiWang) [reply client/src/lib/src/utils/files.ts:15]: One thing we could add is to check if the file has `.bb` before extracting the recipe name.

Comment (deribaucourt) [reply client/src/lib/src/utils/files.ts:15]: This function is sometimes called with gstreamer1.0.bb, sometimes with gstreamer1.0
In which case we both want gstreamer1.0 to be the result. That's why I used explicitely the file extensions like .bb

Comment (deribaucourt) [reply client/src/lib/src/utils/files.ts:16]: Updated the regex to match extensions only at the end of the string

**Review by WilsonZiweiWang — APPROVED** [review]

**Pull request merged**